packetbeat/protos/nfs/request_handler.go

```diff
@@ -76,7 +76,11 @@ func (rpc *Rpc) handleCall(xid string, xdr *Xdr, ts time.Time, tcptuple *common.
 		cred := common.MapStr{}
 		credXdr := Xdr{data: auth_opaque, offset: 0}
 		cred["stamp"] = credXdr.getUInt()
-		cred["machinename"] = credXdr.getString()
+		machnine := credXdr.getString()
+		if len(machnine) == 0 {
```

Comment:
typo: s/machnine/machine/

using `if machine == "" { ... }` is more idiomatic in go.